contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -256,21 +259,22 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     bytes memory response,
     bytes memory err,
     bytes memory onchainMetadata,
-    bytes memory /* offchainMetadata TODO: use in getDonFee() for dynamic billing */
+    bytes memory /* offchainMetadata TODO: use in getDonFee() for dynamic billing */,
+    uint8 reportBatchSize
   ) internal returns (FunctionsResponse.FulfillResult) {
     FunctionsResponse.Commitment memory commitment = abi.decode(onchainMetadata, (FunctionsResponse.Commitment));
 
-    uint96 juelsPerGas = _getJuelsPerGas(tx.gasprice);
+    uint256 gasOverheadWei = (commitment.gasOverheadBeforeCallback + commitment.gasOverheadAfterCallback) * tx.gasprice;
+    uint256 l1FeeShareWei = ChainSpecificUtil._getCurrentTxL1GasFees(msg.data) / reportBatchSize;
```

Comment:
It might warrant a comment explaining why we don't split gasOverheads across all batched requests like we do with the L1 fee.

Reply:
I think that's a good idea. Let's get this one in and then I'll add it in the OCR PR